pkg/controller/controller_utils.go

```diff
@@ -289,21 +315,21 @@ func (s activePods) Less(i, j int) bool {
 	return false
 }
 
-// overlappingControllers sorts a list of controllers by creation timestamp, using their names as a tie breaker.
-type overlappingControllers []api.ReplicationController
+// OverlappingControllers sorts a list of controllers by creation timestamp, using their names as a tie breaker.
+type OverlappingControllers []api.ReplicationController
```

Comment:
I've moved overlapping containers to replication/replication_controller_utils.go